src/Microsoft.ML.DataView/KeyDataViewType.cs

```diff
@@ -3,55 +3,73 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using Microsoft.ML.Internal.Utilities;
-using Microsoft.ML.Runtime;
 
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// KeyTypes are for "id"-like data. The information happens to be stored in an unsigned integer
+    /// KeyDataViewTypes are for "id"-like data. The information happens to be stored in an unsigned integer
     /// type, but the information is not inherently numeric, so, typically, arithmetic is not
     /// meaningful. Examples are SSNs, phone numbers, auto-generated/incremented key values,
     /// class numbers, etc. For example, in multi-class classification, the label is typically
-    /// a class number which is naturally a KeyType.
+    /// a class number which is naturally a KeyDataViewType.
     ///
-    /// KeyTypes have a cardinality (i.e., <see cref="Count"/>) that is strictly positive.
+    /// KeyDataViewTypes have a cardinality (i.e., <see cref="Count"/>) that is strictly positive.
     ///
     /// Note that the underlying representation value does not necessarily match the logical value.
-    /// For example, if a KeyType has range 0-5000, then it has a <see cref="Count"/> of 5001, but
+    /// For example, if a KeyDataViewType has range 0-5000, then it has a <see cref="Count"/> of 5001, but
```

Comment:
>KeyDataViewType [](start = 26, length = 15)

Eh... once we freeze for 1.0 we really ought to work on the documentation. These things really ought to be `<see`, but not the priority right now. Just registering the opinion somewhere.